### DISCUSSION TITLE
WiX Online Meeting 255 - WiX v4 Launch Review on April 4

### DISCUSSION THREAD

**Discussion by robmen** [wixtoolset/issues#7318] [Announcements]
The next WiX Online meeting will be on April 4th, 2023 @9:30 AM Pacific.

Agenda items:

* Triage
* WiX v4 Launch Review

Join us live or later at: https://www.youtube.com/@wixtoolset/

**Comment by chrpai** [comment]
I'm sorry I haven't made the last few meetings. My father in law passed away, my band had it's trailer stolen and a client has been keeping me very busy.  I did just migrate a few blocked projects to rc4 and everything seemed ok.  I've got a bunch more to migrate and will file issues if I find any.  If it's ready, I like the idea of WiX shipping on April 5 as that would be the 9 year anniversary date of MSFT releasing WiX as FOSS.

**Comment by robmen** [comment]
Meeting highlights and video can be found at https://www.firegiant.com/blog/2023/4/4/wix-online-meeting-255-highlights/
